admin/angel/Core.c

```diff
@@ -230,6 +267,7 @@ void Core_init(struct Allocator* alloc,
     ctx->base = eventBase;
     ctx->ipTunnel = ipTunnel;
     ctx->nc = nc;
+    ctx->fileno = fileno;
```

Comment:
Is this used anywhere ?   FileNo_admin_new() should be FileNo_admin_register() and it should return void.